sdk/eventgrid/Azure.Messaging.EventGrid/api/Azure.Messaging.EventGrid.netstandard2.0.cs

```diff
@@ -73,6 +73,106 @@ public enum ServiceVersion
             V2018_01_01 = 1,
         }
     }
+    public static partial class SystemEventMappings
+    {
+        public const string ACSChatMemberAddedToThreadWithUserEvent = "Microsoft.Communication.ChatMemberAddedToThreadWithUser";
```

Comment:
Feedback from Ted: append "Name" to each of the constants so that it will follow a similar pattern as the types themselves which end with "Data".